src/libs/actions/Session.js

```diff
@@ -96,6 +96,10 @@ function fetchAccountDetails(login) {
                     createAccount(login);
                 }
             }
+            Onyx.merge(ONYXKEYS.ACCOUNT, {error: response.message});
+        })
+        .finally(() => {
+            Onyx.merge(ONYXKEYS.ACCOUNT, {loading: false});
```

Comment:
NAB, might be able to combine these into a single merge in the `finally()` by declaring a 

```js
const account = {};
```
before the API call and then modified in `.then()`

Not sure if that's much of an improvement but I thought it could be an interesting convention to follow